src/SourceBuild/content/test/Microsoft.DotNet.UnifiedBuild.Tests/NugetPackageContentTests.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using NuGet.Common;
+using NuGet.Packaging;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.UnifiedBuild.BaselineTests;
+
+
+[Trait("Category", "SdkContent")]
+public class NugetPackageContentTests : TestBase, IClassFixture<NugetPackageContentTests.Config>
+{
+    public class Config
```

Comment:
I don't feel as though a fixture is necessary here. I'd argue that while a fixture can save storage space, you run the risk of slowing down the execution of your tests. Everything in Config is known at compile time, so I'd use a static Config class over a fixture or, even better, define the variables as members of the test class like in https://github.com/dotnet/installer/blob/aed5a974779bba1b9437f5f12e501d744c6d9aa0/src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSizeTest.cs#L22

Reply:
I'll also add that I got a bit confused here because you have a config class in Config.cs, but this is also a config class.

Reply:
Updated the config values to use static properties rather than fixture classes.